internal/executor/util/kubernetes_object.go

```diff
@@ -66,8 +66,13 @@ func CreateIngress(name string, job *api.Job, pod *v1.Pod, service *v1.Service,
 	})
 
 	rules := make([]networking.IngressRule, 0, len(service.Spec.Ports))
-	tlsHosts := make([]string, 0, len(service.Spec.Ports))
+	tlsHosts := make([]string, 0, len(service.Spec.Ports)+1)
 
+	// First host used for certificate signing, needs to be less than 64 chars long
+	firstHost := fmt.Sprintf("%s.svc", name)
```

Comment:
Looks good but we should make the domain extension configurable.

Currently this assumes certs can be signed for the .svc domain which I don't think is going to be true everywhere
 - Happy for the default to be .svc

Should just be a case of adding another field to executor IngressConfiguration + the executor config file to set the default 
 - You already have access to this struct in this func

Reply:
what would be wrong with using the hostnameSuffix? This is already in the executorIngressConfig